modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Telemetry.scala

```diff
@@ -125,6 +125,7 @@ object Telemetry {
   object Cloud {
     case object Aws extends Cloud
     case object Gcp extends Cloud
+    case object Agnostic extends Cloud
```

Comment:
I checked with @stanch , we'll match what stream-collector [does](https://github.com/snowplow/stream-collector/blob/master/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/telemetry/TelemetryAkkaService.scala#L123), i.e. use `None` instead of `Some(Agnostic)`

Thanks for the heads-up!